src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,78 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  max-width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    text-align: right;
+
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  margin: 8px 0 12px;
```

Comment:
The `justify-content: start;` is the default value for flex containers and doesn't need to be explicitly defined unless you're overriding a previous value.